index.html

```diff
@@ -5,39 +5,167 @@
+          </div>
+          <button type="button" class="button">See Project!</button>
+        </div>
+        <div class="grid-item">
+            <img class="work-section-image" src="ressources/Work-3.png" alt="Picture for the first work of Benjamin Patin">
+            <h2>Tonic</h2>
+            <ul class="client-info">
+              <li class="client-name" id="info-client">CANOPY</li>
+              <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
+              <li class="project-style"id="info-client">Back End Dev</li>
+              <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
+              <li class="project-year"id="info-client">2015</li>
+            </ul>
+            <p class="project-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="language-used">
+              <div class="tags">html</div>
+              <div class="tags">css</div>
+              <div class="tags">javaScript</div>
+          </div>
+          <button type="button" class="button">See Project!</button>
+        </div>
+        <div class="grid-item">
+            <img class="work-section-image" src="ressources/Work-4.png" alt="Picture for the first work of Benjamin Patin">
+            <h2>Multi-Post Stories</h2>
+            <ul class="client-info">
+              <li class="client-name" id="info-client">CANOPY</li>
+              <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
+              <li class="project-style"id="info-client">Back End Dev</li>
+              <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
+              <li class="project-year"id="info-client">2015</li>
+            </ul>
+            <p class="project-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="language-used">
+              <div class="tags">html</div>
+              <div class="tags">css</div>
+              <div class="tags">javaScript</div>
+          </div>
+          <button type="button" class="button">See Project!</button>
+        </div>
+      </div>
 
-        <!-- Add font awesome icons -->
-        <ul class="icon-list">
-          <li><a href="https://facebook.com/benj.p009" target="_blank" rel="noopener" class="fa fa-facebook"></a>
-          <a href="https://instagram.com/benjamin_patin" target="_blank" rel="noopener" class="fa fa-instagram"></a>
-          <a href="https://linkedin.com/in/benjaminpatin" target="_blank" rel="noopener" class="fa fa-linkedin"></a>
-          <a href="https://twitter.com/benjp009" target="_blank" rel="noopener" class="fa fa-twitter"></a>
-          <a href="https://github.com/benjp009" target="_blank" rel="noopener" class="fa fa-github"></a></li>
-        </ul>
+      <!-- The "about me section is here"-->
+      <div class="about-me">
```

Comment:
- [ ] Let's change this `div` for a `section` tag as well. By doing so, you can see that our code describes itself. And the comment explaining that a new section starts here is unnecessary, you can leave an empty line to separate big chunks of code that are unrelated.

- [ ] While we are it, let's try to make our design more similar to the design provided in the template. I can see that the top of your section is on the right track, but the bottom still has some improvements to be done.

Yours: 
![image](https://user-images.githubusercontent.com/79114233/138365577-511bc184-20c3-49a3-a56f-8263c94833e8.png)

Template: 
![image](https://user-images.githubusercontent.com/79114233/138365614-4ad39246-1b07-414d-b371-6b0815abc0fc.png)